server/utils/clockTime.ts

```diff
@@ -10,6 +12,21 @@ export default class ClockTime {
     return ClockTime.isValid(hour, minute, second) ? new ClockTime(hour, minute, second) : null
   }
 
+  static fromTwelveHourComponents(
```

Comment:
Now that we have this, is it worth renaming the other `fromComponents` method to specify what type of arguments it takes for clarity?

Reply:
Sure, added a commit for it.